packages/plugin-core/src/utils/docutil.ts

```diff
@@ -0,0 +1,7 @@
+import { TextDocument } from "vscode";
+
+export class DocUtil {
```

Comment:
since this is not plugin specific, can we move this logic to `common-all`? see [style](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.style.utilities.md#L25:L25) for placing utilities

```ts
  static isSchema(uri: URI) {
    return uri.fsPath.endsWith(".schema.yml");
  }
```

Reply:
Was thinking that `"vscode"`
```
import { TextDocument } from "vscode";
```
Is not a dependency that exists within common-all

Reply:
Moved to common-all